quic/s2n-quic-core/src/frame/dc_stateless_reset_tokens.rs

```diff
@@ -0,0 +1,166 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{frame::ExtensionTag, stateless_reset, varint::VarInt};
+use s2n_codec::{decoder_invariant, decoder_parameterized_value, Encoder, EncoderValue};
+
+const TAG: VarInt = VarInt::from_u32(0xdc0000);
+
+macro_rules! dc_stateless_reset_tokens_tag {
+    () => {
+        0xdc0000u64
+    };
+}
+
+//# DC_STATELESS_RESET_TOKENS Frame {
+//#   Type (i) = 0xdc0000,
+//#   Stateless Reset Tokens [(128)],
```

Comment:
Frames need explicit lengths (see STREAM or CRYPTO). Otherwise you'll end up consuming the entire packet payload.

Reply:
just realized that!

Reply:
how about a `Count` like `ACK` frames have `ACK Range Count (i)`? And count * 16 = length

Reply:
I like `Count`, since it's more semantic to what we're communicating.